### DISCUSSION TITLE
Help connecting to MSSQL database

### DISCUSSION THREAD

**Discussion by AbhijeetDixit** [kkrisz1/staruml-reverse-db#125] [Q&A]
I have an MSSQL database which I am logging in to using the windows domain authentication. The database can be accessed using MS SQL Server Management Studio and other applications. 
When I am trying to connect to the same with staruml-reverse-db, it is repeatedly saying login failed for the user. I verified and used the same credentials and server name as MSSQL Mgmt studio. Is there a way to diagnose what actually is causing the issue? Is there something I need to be mindful of to prevent such a failure?
Let me know if further information is required.

**Comment by kkrisz1** [comment]
Currently it is not supported.

Issue #126  is opened

**Reply by kkrisz1** [reply]
could you try the latest release ([v2.1.8](https://github.com/kkrisz1/staruml-reverse-db/releases/tag/v2.1.8)), pls?
